Gemfile.lock

```diff
@@ -312,6 +312,8 @@ GEM
       optimist (~> 3.0)
     pg (1.5.6)
     prometheus-client (2.1.0)
```

Comment:
still has prometheus-client - is this a dependency?

Reply:
exactly - sidekiq_prometheus has a dependency on prometheus-client